src/gui/dialogs/drop_down_menu.cpp

```diff
@@ -139,8 +139,7 @@ void drop_down_menu::pre_show(window& window)
 		find_widget<toggle_panel>(&new_row, "panel", false).set_tooltip(entry["tooltip"]);
 
 		if(entry.has_attribute("image")) {
-			image* img = new image();
-			img->set_definition("default");
+			image* img = build_single_widget_instance<image>("image");
```

Comment:
FTr  the definition is set to "default" if not specified otherwise. As for not making these config arguments, no real reason. I could, I suppose.